frontend/sw360-portlet/src/main/webapp/js/main.js

```diff
@@ -240,9 +240,11 @@ function deleteConfirmed(confirmMessage, confirmCallback) {
         content: confirmMessage,
         confirmButtonClass: 'btn-info',
         cancelButtonClass: 'btn-danger',
-        confirm: function() {
-            confirmCallback();
-        }
+        buttons: {
```

Comment:
The dialog appears when you try to delete a project, a component, or a release, for example. Search the codebase for calls of `deleteConfirmed()` to find more usages.